### DISCUSSION TITLE
Support for checkpoints?

### DISCUSSION THREAD

**Discussion by hugohp** [autodiff/autodiff#279] [Q&A] [answered]
It seems autodiff does not support checkpoint. Any plans to add checkpoint support to reverse mode and if not, why?

**Comment by allanleal** [comment, marked as answer]
Hi @hugohp , there are no plans to develop checkpoints. autodiff is a side product I need for the main library I develop, Reaktoro (https://reaktoro.org). For now, Reaktoro just needs forward mode automatic differentiation, and that has been the priority when developing autodiff. However, everyone is welcome to contribute new features and capabilities to this open source project (and the reverse algorithm has been extended and improved over time due to contributors).

**Reply by hugohp** [reply]
Thanks!
